taxi/views.py

```diff
@@ -1,47 +1,55 @@
+from django.contrib.auth import authenticate, login, logout
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import HttpResponse, HttpRequest, HttpResponseRedirect
```

Comment:
Do not leave imported but not used entities in the future, remove unused imports from the final version

Reply:
Thanks, i always forget smth, my bad.